src/dbt/kipptaf/models/reporting/intermediate/int_reporting__student_filters.sql

```diff
@@ -148,6 +148,55 @@ with
             {{ ref("stg_deanslist__roster_assignments") }} as a
             on r.roster_id = a.dl_roster_id
         where r.school_id = 472 and r.roster_type = 'House' and r.active = 'Y'
+    ),
+
+    state_assessment_exempt as (
+        select
+            'ELA' as discipline,
+
+            co.academic_year,
+            co.student_number,
+
+            case
+                when
+                    co.grade_level < 3
+                    and co.is_self_contained
+                    and co.special_education_code in ('CSE', 'CMO', 'CMI')
+                then true
+                when co.grade_level >= 4 and nj.state_assessment_name in ('2', '3', '4')
+                then true
+                else false
+            end as is_exempt,
+        from {{ ref("base_powerschool__student_enrollments") }} as co
+        left join
+            {{ ref("stg_powerschool__s_nj_stu_x") }} as nj
+            on co.students_dcid = nj.studentsdcid
+            and {{ union_dataset_join_clause(left_alias="co", right_alias="nj") }}
+        where student_number = 100949
+
+        union all
+
+        select
+            'Math' as discipline,
+
+            co.academic_year,
+            co.student_number,
+
+            case
+                when
+                    co.grade_level < 3
+                    and co.is_self_contained
+                    and co.special_education_code in ('CSE', 'CMO', 'CMI')
+                then true
+                when co.grade_level >= 4 and nj.math_state_assessment_name = '3'
+                then true
+                else false
+            end as is_exempt,
+        from {{ ref("base_powerschool__student_enrollments") }} as co
+        left join
+            {{ ref("stg_powerschool__s_nj_stu_x") }} as nj
+            on co.students_dcid = nj.studentsdcid
+            and {{ union_dataset_join_clause(left_alias="co", right_alias="nj") }}
     )
```

Comment:
this should pull solely from `stg_powerschool__s_nj_stu_x` and then move the case statements to the main select, which is already pulling in `base_powerschool__student_enrollments`

Reply:
changes made